<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QLabel" name="TextLabel1_2">
     <property name="text">
      <string>Language:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbLang">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpMode">
     <property name="title">
      <string>Message window</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QRadioButton" name="optSimple">
        <property name="text">
         <string>&amp;Simple mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optChat">
        <property name="text">
         <string>&amp;Chat mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSaveFont">
        <property name="text">
         <string>&amp;Using last selected font</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpContainer">
     <property name="title">
      <string>Container mode</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QRadioButton" name="optNew">
        <property name="text">
         <string>Open each contact in &amp;new container</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optGroup">
        <property name="text">
         <string>Open one container for &amp;group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optOne">
        <property name="text">
         <string>Open all contacts in &amp;one container</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEnter">
        <property name="text">
         <string>Use &quot;&amp;Enter&quot; for send message</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblCopy1">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spnCopy">
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCopy2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer2_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkOwnerName">
        <property name="text">
         <string>Show own nickname in window title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAvatar">
        <property name="text">
         <string>Show user avatar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkStart">
     <property name="text">
      <string>Start SIM on every &amp;windows startup</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="Spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setSimpleMode(bool)</slot>
  <slot>setChatMode(bool)</slot>
  <slot>setOpenEachContactInContainer(bool)</slot>
  <slot>setOpenGroupInContainer(bool)</slot>
  <slot>setOpenAllContactsInOneContainer(bool)</slot>
 </slots>
</ui>
